keys.go

```diff
@@ -38,6 +38,17 @@ var (
 // PreferReturn designates the value for the "Prefer" header.
 type PreferReturn int
 
+// EncryptionAlgorithm represents the encryption algorithm used for key creation
+type EncryptionAlgorithm string
+
+const (
```

Comment:
Lets also leave these in the `import_token.go` file, since that is where the TKEK / import token functions should mostly be organized.